lib/mayaHydra/mayaPlugin/renderOverride.cpp

```diff
@@ -808,7 +810,7 @@ void MtohRenderOverride::_InitHydraResources(const MHWRender::MDrawContext& draw
     _initializationSucceeded = true;
 }
 
-void MtohRenderOverride::ClearHydraResources()
+void MtohRenderOverride::ClearHydraResources(bool fullReset)
```

Comment:
Would be very valuable to have a code comment explaining how to use this new argument.